### PR TITLE
Fix disabled WebView Cookie Tests

### LINKED ISSUE #24913: Fix disabled WebView Cookie Tests
### Description

All of the tests on this test are disabled

https://github.com/dotnet/maui/blob/a7b43dd9e152672b1e7d2ec6d86ddec0c5e35c2c/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue3262.cs#L9

Let's get all of these tests working and passing for each platform in the following order

- Android
- iOS
- Windows

### PR DESCRIPTION
**Pull request by kubaflo**
### Issues Fixed

Fixes https://github.com/dotnet/maui/issues/24913

Screenshot 2024-09-28 at 01 05 47

### CODE REVIEW

**Review by jfversluis — CHANGES_REQUESTED** [review]
Looks like there are some related, relevant test failures

**Comment by kubaflo** [comment]
@jfversluis can we try again please?

**Comment by kubaflo** [comment]
@jfversluis some android tests started to pass, all of them passed on iOS. I have a suspicion that LoadingPageWithoutCookiesSpecifiedDoesntCrash test might be a problem 🤔

**Comment by kubaflo** [comment]
Screenshot 2024-09-30 at 13 30 43

Screenshot 2024-09-30 at 15 03 27

Some progress. I'm not sure why LoadingPageWithoutCookiesSpecifiedDoesntCrash doesn't work though. And why Windows doesn't cooperate

**Comment by kubaflo** [comment]
Let's try with the /azp again :)

File: src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs [hunk 1]
```diff
@@ -13,19 +13,16 @@ public class Issue3262 : TestContentPage // or TestFlyoutPage, etc ...
 
 		protected override void Init()
```
Comment (jsuarezruiz) [inline src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs:14]: The build is failing:
```
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,16): error CS0246: The type or namespace name 'Windows' could not be found (are you missing a using directive or an assembly reference?) [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-maccatalyst]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,5): error CS1929: 'IPlatformElementConfiguration<Windows, WebView>' does not contain a definition for 'SetIsJavaScriptAlertEnabled' and the best extension method overload 'WebView.SetIsJavaScriptAlertEnabled(IPlatformElementConfiguration<Windows, WebView>, bool)' requires a receiver of type 'Microsoft.Maui.Controls.IPlatformElementConfiguration<Microsoft.Maui.Controls.PlatformConfiguration.Windows, Microsoft.Maui.Controls.WebView>' [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-maccatalyst]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,16): error CS0246: The type or namespace name 'Windows' could not be found (are you missing a using directive or an assembly reference?) [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-tizen]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,5): error CS1929: 'IPlatformElementConfiguration<Windows, WebView>' does not contain a definition for 'SetIsJavaScriptAlertEnabled' and the best extension method overload 'WebView.SetIsJavaScriptAlertEnabled(IPlatformElementConfiguration<Windows, WebView>, bool)' requires a receiver of type 'Microsoft.Maui.Controls.IPlatformElementConfiguration<Microsoft.Maui.Controls.PlatformConfiguration.Windows, Microsoft.Maui.Controls.WebView>' [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-tizen]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,16): error CS0246: The type or namespace name 'Windows' could not be found (are you missing a using directive or an assembly reference?) [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-ios]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,5): error CS1929: 'IPlatformElementConfiguration<Windows, WebView>' does not contain a definition for 'SetIsJavaScriptAlertEnabled' and the best extension method overload 'WebView.SetIsJavaScriptAlertEnabled(IPlatformElementConfiguration<Windows, WebView>, bool)' requires a receiver of type 'Microsoft.Maui.Controls.IPlatformElementConfiguration<Microsoft.Maui.Controls.PlatformConfiguration.Windows, Microsoft.Maui.Controls.WebView>' [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-ios]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,16): error CS0246: The type or namespace name 'Windows' could not be found (are you missing a using directive or an assembly reference?) [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-android]
/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs(47,5): error CS1929: 'IPlatformElementConfiguration<Windows, WebView>' does not contain a definition for 'SetIsJavaScriptAlertEnabled' and the best extension method overload 'WebView.SetIsJavaScriptAlertEnabled(IPlatformElementConfiguration<Windows, WebView>, bool)' requires a receiver of type 'Microsoft.Maui.Controls.IPlatformElementConfiguration<Microsoft.Maui.Controls.PlatformConfiguration.Windows, Microsoft.Maui.Controls.WebView>' [/Users/builder/azdo/_work/1/s/src/Controls/tests/TestCases.HostApp/Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-android]
    0 Warning(s)
    8 Error(s)
```
Could you review it?

**Review by jsuarezruiz — CHANGES_REQUESTED** [review]

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue3262.cs [hunk 2]
```diff
@@ -36,12 +37,12 @@ public void ChangeDuringNavigating()
 		}
 
 		[Test]
-		[Category(UITestCategories.WebView)]
 		[Category(UITestCategories.Compatibility)]
-		[FailsOnAllPlatformsWhenRunningOnXamarinUITest]
+		[FailsOnMac]
```
Comment (jsuarezruiz) [inline src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue3262.cs:41]: Needs also to update this attributes, have changed.
`[FailsOnMacWhenRunningOnXamarinUITest]`

File: src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs (hunk 1 above)
Comment (kubaflo) [reply src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs:15]: Should work now

**Review by jfversluis — CHANGES_REQUESTED** [review]
Build error

> D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue3262.cs(48,16): error CS0118: 'Windows' is a namespace but is used like a type [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.19041.0]
D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue3262.cs(48,5): error CS1929: 'IPlatformElementConfiguration<Windows, WebView>' does not contain a definition for 'SetIsJavaScriptAlertEnabled' and the best extension method overload 'WebView.SetIsJavaScriptAlertEnabled(IPlatformElementConfiguration<Windows, WebView>, bool)' requires a receiver of type 'Microsoft.Maui.Controls.IPlatformElementConfiguration<Microsoft.Maui.Controls.PlatformConfiguration.Windows, Microsoft.Maui.Controls.WebView>' [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.19041.0]
D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue3262.cs(48,16): error CS0118: 'Windows' is a namespace but is used like a type [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.20348.0]
D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue3262.cs(48,5): error CS1929: 'IPlatformElementConfiguration<Windows, WebView>' does not contain a definition for 'SetIsJavaScriptAlertEnabled' and the best extension method overload 'WebView.SetIsJavaScriptAlertEnabled(IPlatformElementConfiguration<Windows, WebView>, bool)' requires a receiver of type 'Microsoft.Maui.Controls.IPlatformElementConfiguration<Microsoft.Maui.Controls.PlatformConfiguration.Windows, Microsoft.Maui.Controls.WebView>' [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.20348.0]

**Comment by kubaflo** [comment]
> Build error
> 
> > D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue3262.cs(48,16): error CS0118: 'Windows' is a namespace but is used like a type [D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.19041.0]
> > D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue3262.cs(48,5): error CS1929: 'IPlatformElementConfiguration<Windows, WebView>' does not contain a definition for 'SetIsJavaScriptAlertEnabled' and the best extension method overload 'WebView.SetIsJavaScriptAlertEnabled(IPlatformElementConfiguration<Windows, WebView>, bool)' requires a receiver of type 'Microsoft.Maui.Controls.IPlatformElementConfiguration<Microsoft.Maui.Controls.PlatformConfiguration.Windows, Microsoft.Maui.Controls.WebView>' [D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.19041.0]
> > D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue3262.cs(48,16): error CS0118: 'Windows' is a namespace but is used like a type [D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.20348.0]
> > D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue3262.cs(48,5): error CS1929: 'IPlatformElementConfiguration<Windows, WebView>' does not contain a definition for 'SetIsJavaScriptAlertEnabled' and the best extension method overload 'WebView.SetIsJavaScriptAlertEnabled(IPlatformElementConfiguration<Windows, WebView>, bool)' requires a receiver of type 'Microsoft.Maui.Controls.IPlatformElementConfiguration<Microsoft.Maui.Controls.PlatformConfiguration.Windows, Microsoft.Maui.Controls.WebView>' [D:\a_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.20348.0]

Hmm not sure what the problem is. I added a commit that may fix it

File: src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs [hunk 3]
```diff
@@ -1,7 +1,5 @@
 ﻿using System.Net;
 using System.Text.RegularExpressions;
-using Microsoft.Maui.Controls.PlatformConfiguration;
-using Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific;
```
[the remark this answers is not in the document]
Comment (kubaflo) [reply src/Controls/tests/TestCases.HostApp/Issues/Issue3262.cs:3]: Thanks

**Review by jfversluis — APPROVED** [review]

**Review by jsuarezruiz — APPROVED** [review]

**Pull request merged**